services/baseService.go

```diff
@@ -0,0 +1,23 @@
+package controllers
+
+import (
+	"net/http"
+
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/datastore"
+)
+
+type Entity interface {
+	SetID(id string)
+}
+
+func SaveObject(w http.ResponseWriter, r *http.Request, entity Entity, entityKind string) error {
```

Comment:
This is not used anywhere. I am not sure how you plan to use it, so please leave this change out for now, and add it once you really make use of the functionality.